.github/workflows/build.yml

```
@@ -0,0 +1,14 @@
+on: [push, fork, pull_request]
+jobs:
+  lint-test-build-package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
```

```suggestion
          go-version: 1.19
          cache: true
```

Caching seems to be disabled by default.

---

added